src/main/java/teammates/logic/core/GoogleCloudLoggingService.java

```diff
@@ -209,20 +283,95 @@ private void createLogEntry(LogEntry entry) throws LogServiceException {
         return logEntries;
     }
 
+    private Page<LogEntry> getLogEntriesByPage(LogSearchParams s, PageParams p) throws LogServiceException {
```

Comment:
This method is to get `LogEntry` with pagination related parameters and return the `LogEntry` in `Page` instead of in `List`. The methods for getting the `LogEntry` in a `Page` is also different. Combining into one may cause result in many if else branches. I can try combining.

Reply:
I have combine the two methods into one. The `getLogEntries` method now returns `Page<LogEntry>` and the code for converting From Page to List has been moved to `getRecentErrorLogs` and `getFeedbackSessionLogs`.